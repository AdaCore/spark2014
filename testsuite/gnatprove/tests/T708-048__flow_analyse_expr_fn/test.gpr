project Test is
  --  Deliberately empty
end Test;
