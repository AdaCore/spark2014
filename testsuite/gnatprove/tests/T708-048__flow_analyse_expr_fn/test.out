tcp_type.ads:21:13: info: implicit terminating annotation on "Tcp_Syn_Queue_Item_Model" has been proved, subprogram will terminate
tcp_type.ads:24:10: info: subprogram variant proved (Trivial)
tcp_type.ads:24:42: info: pointer dereference check proved (CVC5: 1 VC)
tcp_type.ads:25:21: warning: Always_Return annotation is useless
tcp_type.ads:25:21: warning: function "Tcp_Syn_Queue_Item_Model" has implicit Always_Return annotation
